packages/react-charts/src/LineGraph/index.tsx

```diff
@@ -107,17 +107,25 @@ const LineGraph = (props: LineGraphProps) => {
   };
 
 
+  const getIndexFromScaledPoint = (x:number, series: Point[]) => {
+    const bisectXAxis = bisector(getXaxisValue).left;
+
+    return bisectXAxis(series, invertX(x), 0);
+  };
+
+
   const getDragColor = (lp: LinePathData) => {
-    const { startIndex, endIndex } = dragPoints ?? {};
+    const { startPoint, endPoint } = dragPoints ?? {};
     const { fill = lp?.color, lineColor, negativeLineColor } = lp?.draggableConfig ?? {};
 
-    if (startIndex === undefined || endIndex === undefined) return lp?.color;
-    const lpStart = lp?.series?.[startIndex]?.[1];
-    const lpEnd = lp?.series?.[endIndex]?.[1];
+    if (startPoint === undefined || endPoint === undefined) return lp?.color;
```

Comment:
You can use isEmpty here

Reply:
Why to use isEmpty if I don't want to have multiple checks except undefined?